pkg/cmd/orders.go

```diff
@@ -0,0 +1,65 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+// go run ./cmd/bbgo orders [open|closed] --session=ftx --symbol=BTC/USDT
+var ordersCmd = &cobra.Command{
+	Use:  "orders [status]",
+	Args: cobra.OnlyValidArgs,
+	// default is open which means we query open orders if you haven't provided args.
+	ValidArgs:    []string{"", "open", "closed"},
+	SilenceUsage: true,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		ctx := context.Background()
+		session, err := cmd.Flags().GetString("session")
+		if err != nil {
+			return fmt.Errorf("can't get session from flags: %w", err)
+		}
+		ex, err := newExchange(session)
```

Comment:
you can try using the environment to get the exchange later (see pnl command for the example)

Reply:
Do you mean get credential? If that, ftx not only need key and secret. It may also need your account name if your credential is from subaccount.